src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -58,19 +57,9 @@ import { WithdrawContext } from '../WithdrawContext'
 type WithdrawProps = StepComponentProps & { accountId: AccountId | undefined }
 
 export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
-  const [maybeOutboundFeeCryptoBaseUnit, setMaybeOutboundFeeCryptoBaseUnit] = useState<Result<
-    string,
-    string
-  > | null>()
-  const [maybeWithdrawGasEstimateCryptoBaseUnit, setMaybeWithdrawGasEstimateCryptoBaseUnit] =
-    useState<Result<string, string> | null>(null)
   const [slippageCryptoAmountPrecision, setSlippageCryptoAmountPrecision] = useState<string | null>(
     null,
   )
-  const [inputValues, setInputValues] = useState<{
-    fiatAmount: string
-    cryptoAmount: string
-  } | null>(null)
```

Comment:
This had no use at all, it's effectively doing the `getValues()` dance, setting the *exact* same values, not even a mask, in a badly reactive way and was causing us a whole pack of issues including validation actually not working despite debugging it and everything looking seemingly good under the hood.

Pls accept 6e2355bb58aaa58185d0e6d2171c743303de67a3 as my cathartic contribution